src/parser/types.ts

```diff
@@ -568,10 +776,118 @@ export interface RepeatedComponentBlockCondition {
   to: string;
 }
 
-/** The SkipConditions interface is used to define skip conditions. This is used to skip to a different component or block based on the response to a component or the number of correct or incorrect responses in a block. Skip conditions work recursively, that is if you have a nested block, they parent blocks' skip conditions will be considered when computing the skip logic. */
+/** The SkipConditions interface is used to define skip conditions. This is used to skip to a different component or block based on the response to a component or the number of correct or incorrect responses in a block. Skip conditions work recursively, that is if you have a nested block, the parent blocks' skip conditions will be considered when computing the skip logic.
+ *
+ * Skip conditions are evaluated in the order they are defined in the array. If a condition is met, the participant will be redirected to the component or block specified in the to property. If no conditions are met, the participant will continue to the next component in the sequence.
+ *
+ * Skip conditions allow you to jump to a different component or block. If you intend to skip to a block, you should specify a block id in the sequence. If you intend to skip to a component, you should specify a component id. Skipping backwards is not supported. Skipping to a repeated component will skip to the first instance of the component after the component that triggered the skip.
+ *
+ * Please see the interface definitions for more specific information on the different types of skip conditions.
+*/
 export type SkipConditions = (IndividualComponentSingleResponseCondition | IndividualComponentAllResponsesCondition | ComponentBlockCondition | RepeatedComponentBlockCondition)[];
 
-/** The ComponentBlock interface is used to define order properties within the sequence. This is used to define the order of components in a study and the skip logic. It supports random assignment of trials using a pure random assignment and a latin square. */
+/** The ComponentBlock interface is used to define order properties within the sequence. This is used to define the order of components in a study and the skip logic. It supports random assignment of trials using a pure random assignment and a [latin square](https://en.wikipedia.org/wiki/Latin_square).
+ *
+ * The pure random assignment is a random assignment with no guarantees. For example, it's possible, though unlikely, that one component would show up in the first position 10 times in a row. Here's a snippet that shows how to use the random order:
```

Comment:
```suggestion
 * The pure random assignment is a random assignment with no guarantees. For example, one component _could_ show up in the first position 10 times in a row. However, this situation is unlikely.
 
 Here's a snippet that shows how to use the random order:
```
Not sure I like how I re-wrote this. But the "For example, it's possible, though unlikely, ... " felt clunky.